apps/automation-v2/src/config/prodnets.ts

```diff
@@ -0,0 +1,23 @@
+import { parseEther } from 'ethers';
+import { Chain, MAINNET } from '@/chains';
+import { CheckEthBalanceType } from '@/common/eth-auto-checker';
+import { TlcBatchLiquidateConfig } from '@/tlc/batch-liquidate';
+
+const CHECK_ETH_BALANCE_CONFIG: CheckEthBalanceType = (chain: Chain) => ({
+  CHAIN: chain,
+  WALLET_NAME: 'temple_automation',
+  MIN_ETH_BALANCE: parseEther('0.1'),
+});
+
+const TLC_BATCH_LIQUIDATE_CONFIG: TlcBatchLiquidateConfig = {
+  CHAIN: MAINNET,
+  WALLET_NAME: 'temple_automation',
+  TLC_ADDRESS: '0xcbc0A8d5C7352Fe3625614ea343019e6d6b89031',
+  ACC_LIQ_MAX_CHUNK_NO: 100,
+  MIN_ETH_BALANCE: parseEther('0.05'),
```

Comment:
Does this mean we check the ETH balance both on a schedule and after the liquidate? or is this something different?

Reply:
yep, we check eth balance every 30 min, but also we check balance in after each batch liquidate chunk in this task, took the idea from https://github.com/TempleDAO/temple/pull/861#discussion_r1337866952

Reply:
I don't really see the value in checking the eth balance every 30 mins.  

Nothing except the ramos automation task can spend money from the `ramos_automation` wallet (0x67ae2161449cf3c5528fea969beba3e4f3288c61).

Reply:
sorry - this is not ramos, but the point remains. We will deploy a wallet specifically for these automations (to avoid nonce concurrency constraints), so no need to check except after tlc liquidate operations?

Reply:
I guess this is just an extra check to inform the team in case the wallet has been used somewhere else, but... if wallet is specific for this, then no need, happy to remove if you think that's better/clearer